UI/SidePanel.cpp

```diff
@@ -3211,7 +3211,9 @@ void SidePanel::DoLayout() {
 
     // system name / droplist
     ul = GG::Pt(GG::X(MaxPlanetDiameter()), GG::Y0);
-    lr = ul + GG::Pt(ClientWidth() - GG::X(MaxPlanetDiameter()), name_height);
+    auto system_name_width = ClientWidth() - GG::X(MaxPlanetDiameter());
```

Comment:
I think implicit casts to X are disabled?

I question whether the supposed "future proofing" would really be of much help, compared with the clarity of explicitly stating the type in such cases, but OK...